packages/core/test/node/env-dependencies.spec.ts

```diff
@@ -0,0 +1,93 @@
+import chai, { expect } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import sinonChai from 'sinon-chai';
+import { Environment, Feature, run as runEngine, Service } from '@wixc3/engine-core';
+
+chai.use(sinonChai);
+chai.use(chaiAsPromised);
+
+describe('ENV dependencies', () => {
+    it('simple env dependency', async () => {
+        const baseEnv = new Environment('baseEnv', 'node', 'multi');
```

Comment:
an error should be thrown (and displayed maybe type-wise) if extendingEnv is not multi, or if it's node node